distributions/distribution-resources/src/main/resources/deb/systemd/openhab2.service

```diff
@@ -1,6 +1,7 @@
 [Unit]
-Description=Starts and stops the openHAB Home Automation Bus
-Documentation=http://www.openhab.org
+Description=openHAB 2 - empowering the smart home
+Documentation=https://docs.openhab.org
```

Comment:
I'm able to reach https://docs.openhab.org but I just saw that it seems like I have added an exception to my browser to allow the mismatched certificate. fixing now.